File: AndroidStudioProjects/Demo/app/src/main/java/com/test/demo/glide/TestActivity.java
package com.test.demo.glide;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.test.demo.R;

public class TestActivity extends AppCompatActivity implements View.OnClickListener{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()){
            case R.id.text:
                startActivity(new Intent(TestActivity.this, GifTestActivity.class));
                break;
        }
    }
}

File: AndroidStudioProjects/Kotlin/app/src/main/java/com/test/kotlin/TestActivity.kt
package com.test.kotlin

import android.graphics.Rect
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import kotlinx.android.synthetic.main.activity_test.*

class TestActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)
        text.text = Test("1").test().toString();
        var num = arrayListOf<Rect>();
        var strs:String = "";
    }

}
